docs/komodo/setup-electrumX-server.md

```diff
@@ -1,103 +1,164 @@
 # Setup ElectrumX Server
 
-Here are the steps required to run electrumx for KMD. Replace the variables according to your setup. You need to setup minimum 2 electrum servers for same coin to ensure stable operation.
+An SPV Electrum Server is a server that provides "lite mode" type functionality. These servers run a full node, which maintains a copy of a blockchain's history (a.k.a.) "native mode". The electrum software allows developers and apps like AtomicDEX to communicate with the blockchain to query balances, transaction history or broadcast signed transactions without needing to download and sync the full chain locally.
 
-Refer to the docs at [https://electrumx.readthedocs.io/en/latest/](https://electrumx.readthedocs.io/en/latest/) for more info.
+A list of known electrum servers is maintained at https://github.com/KomodoPlatform/coins/tree/master/electrums
+It's recommended to run at least 2 - 3 electrum servers to ensure stable operation.
 
-- `<username>` is the username under which electrumx will run
+Check out the [ElectrumX Docs](https://electrumx.readthedocs.io/en/latest/) for more info.
 
-- `$rpcuser` and `$rpcpass` are from the conf of the wallet daemon
 
-## General part
+## Installation
 
 ```bash
 sudo apt-get install python3-setuptools python3-multidict python3.8 libleveldb-dev
-cd /home/<username>
+cd ~
 git clone https://github.com/KomodoPlatform/electrumx-1
-cd electrumx-1
+cd ~/electrumx-1
 pip3 install .
 pip3 install .[uvloop,ujson]
 ```
 
-## Coin specific part
 
-If you are launching the electrum server for a new smartchain, you will have to add it to the `electrumx-1/electrumx/lib/coins.py` file
+## Coin Configuration
 
-For example: 
+If you are launching the electrum server for a new smartchain, you will have to add it to the `~/electrumx-1/electrumx/lib/coins.py` file
```

Comment:
should we add a new doc that describes creating a new Electrum server for KMD  itself? or a note in this one that explains the differences should be enough?